src/main/java/bio/terra/testrunner/runner/UserJourneyResult.java

```diff
@@ -19,6 +19,8 @@
   public String exceptionStackTrace;
   public String exceptionMessage;
 
+  public UserJourneyResult() {}
```

Comment:
Just curious, why was this needed?

Reply:
I think that's because `jackson` complained about the need of the no-arg constructor when I added `public TestScriptUserJourneySnapshots userJourneySnapshots` to `TestScriptResult` class.